### PR TITLE
Review of api connector commands

### PR DESCRIPTION
**Pull request by johnfolly**
- Update code format based on `.style.yapf` 
- Added connector get-currents : retrived adt_connector and storage_connector details of babylon deploy
- change import format to relative

```bash
echo "Babylon api connector command [Common usage]"
# Get all registered connectors
babylon api connector get-all
# get all registered connectors and filter fields to retrieve only id and name
babylon api connector get-all -f "id, name"
# Get all registered connectors and dump them in a file
babylon api connector get-all -o ../test.json
# get all registered connectors, filter fields to retrieve only id and name and dump them in a file
babylon api connector get-all -f "id, name" -o ../test.json
# create an connector
babylon api connector create "Babylon test ADT Connector" --version="2.4.0" --type=ADT --output-file=/tmp/connector_test.json
babylon api connector create "Babylon test STORAGE Connector" --version="2.4.0" --type=STORAGE --output-file=/tmp/connector_test.json
# get the current connector set in babylon deploy config
babylon api connector get-currents
# get the current connector set in babylon deploy config and filter fields to retrieve only id and name
babylon api connector get-currents -f "id, name"
# dump the the current connector set in babylon deploy in json file
babylon api connector get-currents -o ../test.json
# get the the current connector set in babylon deploys, filter fields to retrieve only id and name name and dump them in a file
babylon api connector get-currents -f "id, name" -o ../test.json
# delete the created connector
babylon api connector delete -f -i /tmp/connector_test.json
# get all registered connectors and filter fields to retrieve only id and name
babylon api connector get-all -f "id, name"
```

- Get command work as get-currenrt 
```bash
# get the created connector details
babylon api connector get c-anConnectorId
# dump the created connector details in json file
babylon api connector get c-anConnectorId -o ../test.json
# update the created connector
babylon api connector update ../test.json
# dump the created connector details and filter fields to retrieve only id and name
babylon api connector get c-anConnectorId -f "id, name"
# dump the created connector details in json file
babylon api connector get c-anConnectorId -o ../test.json
# get the created connector details, filter fields to retrieve only id and name name and dump them in a file
babylon api connector get c-anConnectorId -f "id, name" -o ../test.json
```

### CODE REVIEW

**Comment by o-reo** [comment]
Hi !
You have to update the config templates located in tests, it's not ideal I will have to change this.

File: Babylon/groups/api/groups/connector/commands/get_currents.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+@pass_connector_api
+@option("-o",
+        "--output_file",
+        "output_file",
+        help="The path to the file where Connectors should be outputted (json-formatted)",
+        type=Path())
+@option("-f",
+        "--fields",
+        "fields",
+        required=False,
```
Comment (o-reo) [inline Babylon/groups/api/groups/connector/commands/get_currents.py:38]:
```suggestion
```
This is redundant, you can remove this line

File: Babylon/groups/api/groups/connector/commands/get_currents.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        type=str,
```
Comment (o-reo) [inline Babylon/groups/api/groups/connector/commands/get_currents.py:39]:
```suggestion
```

**Review by o-reo — COMMENTED** [review]
Minor coding style feedback, thanks for your PR

**Comment by o-reo** [comment]
If you rebase you no longer working dir test files !

**Comment by johnfolly** [comment]
> If you rebase you no longer working dir test files !

Thank, it better now

File: tests/resources/workingdir/API/Connector.ADT.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      - id: AZURE_DIGITAL_TWINS_URL
+        label: Azure Digital Twins URL
+        valueType: string
+        envVar: AZURE_DIGITAL_TWINS_URL
+      - id: CSM_NUMBER_OF_THREAD
+        label: Number of thread used
+        valueType: int
+        options: null
+        default: '1'
+        envVar: CSM_NUMBER_OF_THREAD
```
Comment (o-reo) [inline tests/resources/workingdir/API/Connector.ADT.yaml:26]: No longer necessary you can delete this file

File: terraform_cloud.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# token : API token to connect to terraform cloud API
+# can be created in this page : https://app.terraform.io/app/settings/tokens
+token: "SECURITY_TOKEN"
+# api : URL of the API for terraform cloud
+url: https://app.terraform.io
+# organization : organization to use on terraform cloud
+organization: "ORGANIZATION"
```
Comment (o-reo) [inline terraform_cloud.yaml:7]: You should remove this file

File: terraform_workspace.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# workspace_id: id of the workspace to use by default
+workspace_id: ""
+# working_directory: working directory of the workspace
+working_directory: ""
+# vcs_identifier: identifier of the vcs associated to the workspace
+vcs_identifier: ""
+# vcs_branch: branch in the vcs used by the workspace
+vcs_branch: ""
+# vcs_oauth_token_id: id to the oauth token used to connect to the vcs
+vcs_oauth_token_id: ""
```
Comment (o-reo) [inline terraform_workspace.yaml:10]: You should remove this file

File: API/Dataset.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - ADT
+  - ###
+  - Reference
+  - dataset
+connector:
+  id: ###
+  parametersValues:
+    AZURE_DIGITAL_TWINS_URL: ###
+    CSM_NUMBER_OF_THREAD: 1
```
Comment (o-reo) [inline API/Dataset.yaml:13]: You should remove this file

File: API/Solution.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    applyParameters: true
+    sendDatasetsToDataWarehouse: true
+    sendInputParametersToDataWarehouse: false
+    validateData: false
+    preRun: false
+    run: true
+    postRun: false
+    parametersJson: true
+    parameterGroups:
+      - parameter_group1
```
Comment (o-reo) [inline API/Solution.yaml:47]: You should remove this file

**Review by o-reo — COMMENTED** [review]
Just some files to remove :+1:

**Comment by o-reo** [comment]
Can you delete the workingdir.zip file ?

**Comment by johnfolly** [comment]
> Can you delete the workingdir.zip file ?

Okay 👌

**Comment by o-reo** [comment]
Oh my bad, I thought it was no longer needed but it is !

File: API/Connector.ADT.yaml [hunk 8]
```diff
@@ -1,7 +1,7 @@
-key: ADT Connector
-name: ADT Connector
+key:
+name:
 repository: cosmo-tech/azure-digital-twins-simulator-connector
-version: 2.4.0
+version:
 azureAuthenticationWithCustomerAppRegistration: true
 ioTypes:
   - read
```
Comment (o-reo) [inline API/Connector.ADT.yaml:7]: The file should be deleted

File: API/Connector.STORAGE.yaml [hunk 9]
```diff
@@ -1,8 +1,8 @@
-key: AzureStorageConnector
-name: Azure Storage Connector
+key:
+name:
 description: Connector for Azure Storage. Read all data in a container with a prefix and write the data in CSV for a ScenarioRun
 repository: cosmo-tech/azure-storage-simulator-connector
-version: 1.1.2
+version:
 tags:
   - Azure Storage
```
Comment (o-reo) [inline API/Connector.STORAGE.yaml:7]: The file should be deleted

File: API/Dataset.yaml (hunk 6 above)
Comment (o-reo) [inline API/Dataset.yaml:13]: The file should be deleted

**Review by o-reo — COMMENTED** [review]
Still some unexpected files in the API folder at the root of the project

**Review by o-reo — APPROVED** [review]

**Pull request merged**